### PR TITLE
Retranslate initializers.md

### PR DESCRIPTION
**Pull request by EthanJYK**
[Summary]
- Retranslated overall contexts and tried to paraphrase details as much as possible, especially in order to help newcomers understand how functions work.
- Added missing notations of defalut values.

[For Reviewers Proficient in Korean]
- 직역이 의미를 충분히 전달하지 못하는 경우 의역하여 충분히 설명이 이루어지는 데에 집중했습니다.
- 원문에 설명이 부족한 경우 이해를 도울 수 있는 내용을 추가하였습니다.
- glorot~, he~, lecun~ 계열의 생성 함수는 순서가 뒤섞여있던 것을 glorot, he, lecun 순으로 묶었습니다. 리뷰시 바뀐 순서에 유의 부탁드립니다. 
- 간혹 원문에 weight라고 되어있는 부분이 실제로는 weight와 bias 모두에 적용되는 경우가 있습니다. 이 경우는 원래의 용도를 고려하여 양쪽을 다 가리키는 말인 파라미터로 번역합니다.
- custom -> 사용자 정의
- callable -> 호출 가능한 (함수)
- 그 밖의 용어 번역, 문서 양식은 Keras Korea 번역팀에서 합의된 가이드라인을 따릅니다.

### CODE REVIEW

File: sources/initializers.md [hunk 1]
```diff
@@ -1,394 +1,277 @@
...
-- mode = "fan_avg"의 경우 입력 유닛과 출력 유닛의 수의 평균을 말합니다.
+또는 균등분포(`distribution="uniform"`)를 선택할 경우 `[-limit, limit]`의 범위를 가진 균등분포로부터 값이 선택됩니다. 여기서 `limit`은 `sqrt(3 * scale / n)`으로 정의됩니다. 위와 마찬가지로 `n`값은 `mode`인자에 따라 정해집니다.
 
-`distribution="uniform"`로 분포를 설정 시,
-[-limit, limit]의 범위내 균등분포에 따라 샘플이 생성되는데,
-여기서 limit은 `limit = sqrt(3 * scale / n)`로 정의됩니다.
+__인자__
+- __scale__: 스케일링 팩터(양의 `float`)입니다. 기본값은 `1.0`입니다.
+- __mode__: `"fan_in"`, `"fan_out"`, `"fan_avg"` 중 하나입니다. 기본값은 `"fan_in"`입니다.
+- __distribution__: `"normal"`과 `"uniform"` 가운데에서 분포의 종류를 정합니다. 기본값은 `"normal"`입니다.
```
Comment (airplane2230) [inline sources/initializers.md]: `"normal"`과 `"uniform"` 중에서 분포의 종류를 지정할 수 있습니다.

또는 가운데에서 ~

+ 위와 같이 중에서나 가운데에서, 두 표현 중 하나를 지정하여 통일하는게 어떨까요?

File: sources/initializers.md [hunk 2]
```diff
@@ -1,394 +1,277 @@
...
+  
 ----
-
 ### glorot_normal
-
-
 ```python
 keras.initializers.glorot_normal(seed=None)
 ```
+Glorot 정규분포 방식으로 파라미터의 초기값을 생성합니다. Xavier 정규분포 방식이라고도 불리며, 가중치 텐서의 크기에 따라 값을 조절하는 방식의 하나입니다.
```
Comment (airplane2230) [inline sources/initializers.md]: 가중치 텐서의 크기에 따라 값을 조절하는 방식의 하나입니다. 방식의 하나라고 하면 뭔가 끝에 정리가 되어있어야 할 것 같습니다. 의견 부탁드려요. 일단 나머지도 전부 표기해두도록 하겠습니다.

+ 가중치 텐서의 크기에 따라 값을 조절됩니다.
로 바꾸면 어떨까요.

File: sources/initializers.md [hunk 3]
```diff
@@ -1,394 +1,277 @@
...
     
 ----
-
 ### glorot_uniform
-
-
 ```python
 keras.initializers.glorot_uniform(seed=None)
 ```
+Glorot 균등분포 방식으로 파라미터의 초기값을 생성합니다. Xavier 균등분포 방식이라고도 불리며, 가중치 텐서의 크기에 따라 값을 조절하는 방식의 하나입니다.
```
Comment (airplane2230) [inline sources/initializers.md]: 가중치 텐서 ~ 표현

File: sources/initializers.md [hunk 4]
```diff
@@ -1,394 +1,277 @@
...
     
 ----
-
 ### he_normal
-
-
 ```python
 keras.initializers.he_normal(seed=None)
 ```
+He 정규분포 방식으로 파라미터의 초기값을 생성합니다. 가중치 텐서의 크기에 따라 값을 조절하는 방식의 하나입니다.
```
Comment (airplane2230) [inline sources/initializers.md]: 가중치 텐서 ~ 표현

File: sources/initializers.md [hunk 5]
```diff
@@ -1,394 +1,277 @@
...
-__인수__
+__참조__
+- [Delving Deep into Rectifiers: Surpassing Human-Level Performance on ImageNet Classification](http://arxiv.org/abs/1502.01852)
+
+----
+### he_uniform
+```python
+keras.initializers.he_uniform(seed=None)
+```
+He 정규분포 방식으로 파라미터의 초기값을 생성합니다. 가중치 텐서의 크기에 따라 값을 조절하는 방식의 하나입니다.
```
Comment (airplane2230) [inline sources/initializers.md]: 가중치 텐서 ~ 표현

File: sources/initializers.md [hunk 6]
```diff
@@ -1,394 +1,277 @@
...
-    
 ----
-
 ### lecun_normal
-
-
 ```python
 keras.initializers.lecun_normal(seed=None)
 ```
+LeCun 정규분포 방식으로 파라미터의 초기값을 생성합니다. 가중치 텐서의 크기에 따라 값을 조절하는 방식의 하나입니다.
```
Comment (airplane2230) [inline sources/initializers.md]: 가중치 텐서 ~ 표현

File: sources/initializers.md [hunk 7]
```diff
@@ -1,394 +1,277 @@
...
-### he_uniform
-
 
+----
+### lecun_uniform
 ```python
-keras.initializers.he_uniform(seed=None)
+keras.initializers.lecun_uniform(seed=None)
 ```
+LeCun 정규분포 방식으로 파라미터의 초기값을 생성합니다. 가중치 텐서의 크기에 따라 값을 조절하는 방식의 하나입니다.
```
Comment (airplane2230) [inline sources/initializers.md]: 가중치 텐서 ~ 표현

File: sources/initializers.md [hunk 8]
```diff
@@ -1,394 +1,277 @@
...
+- [Efficient BackProp](http://yann.lecun.com/exdb/publis/pdf/lecun-98b.pdf)
 
-- [Delving Deep into Rectifiers: Surpassing Human-Level Performance on
-   ImageNet Classification](http://arxiv.org/abs/1502.01852)
     
-
-
-초기값 설정기는 문자열(위의 사용가능한 초기값 설정기와 매치되어야 합니다)이나 callable 로 전달할 수 있습니다:
+----
+신경망의 각 층에서 초기값 생성 함수는 해당 초기값 생성 함수의 이름과 동일한 문자열<sub>string</sub> 또는 호출 가능한 함수<sub>callable</sub>의 형태로 지정할 수 있습니다.
```
Comment (airplane2230) [inline sources/initializers.md]: 신경망의 각 케라스 층에서 초기값 생성 함수를 사용할 때, 다음 예시와 같이 해당 초기값 생성 함수의 이름과 동일한 문자열<sub>string</sub> 또는 호출 가능한 함수<sub>callable</sub>의 형태로 지정할 수 있습니다.

File: sources/initializers.md [hunk 9]
```diff
@@ -1,394 +1,277 @@
...
-- mode = "fan_in"의 경우 가중치 텐서의 입력 유닛의 수입니다.
-- mode = "fan_out"의 경우 가중치 텐서의 출력 유닛의 수입니다.
-- mode = "fan_avg"의 경우 입력 유닛과 출력 유닛의 수의 평균을 말합니다.
+또는 균등분포(`distribution="uniform"`)를 선택할 경우 `[-limit, limit]`의 범위를 가진 균등분포로부터 값이 선택됩니다. 여기서 `limit`은 `sqrt(3 * scale / n)`으로 정의됩니다. 위와 마찬가지로 `n`값은 `mode`인자에 따라 정해집니다.
 
-`distribution="uniform"`로 분포를 설정 시,
-[-limit, limit]의 범위내 균등분포에 따라 샘플이 생성되는데,
-여기서 limit은 `limit = sqrt(3 * scale / n)`로 정의됩니다.
+__인자__
+- __scale__: 스케일링 팩터(양의 `float`)입니다. 기본값은 `1.0`입니다.
```
[the remark this answers is not in the document]
Comment (EthanJYK) [reply sources/initializers.md]: '값'으로 가겠습니다.

File: sources/initializers.md [hunk 10]
```diff
@@ -1,394 +1,277 @@
...
+- __mode__: `"fan_in"`, `"fan_out"`, `"fan_avg"` 중 하나입니다. 기본값은 `"fan_in"`입니다.
```
[the remark this answers is not in the document]
Comment (EthanJYK) [reply sources/initializers.md]: ~중에서 지정합니다. 로 수정했습니다.

File: sources/initializers.md (hunk 1 above)
Comment (EthanJYK) [reply sources/initializers.md]: ~ 가운데에서 분포의 종류를 지정합니다. 로 하겠습니다.

File: sources/initializers.md (hunk 2 above)
Comment (EthanJYK) [reply sources/initializers.md]: 입력(혹은 입출력)텐서의 크기에 따라 가중치를 조절하는 방식은 Xavier나 He, LeCun 모두 마찬가지이기 때문에, 개별 메소드들이 하나의 아이디어에서 파생된 여러 방식 가운데 하나라는 뜻을 확실히 할 필요가 있어서 그와 같이 서술했습니다. 문서 전체에서 크기에 따른 조절 방식 여러 개를 다루고 있으므로 별도의 추가 설명은 없어도 될 것 같다고 생각합니다.

File: sources/initializers.md (hunk 8 above)
Comment (EthanJYK) [reply sources/initializers.md]: "케라스 모델의 각 층에 초기값 생성 함수를 적용하는 방법은 크게 두 가지입니다. 먼저 케라스가 제공하는 함수인 경우 `(kernel, bias 등)_initializer` 인자에 초기값 생성 함수의 이름과 동일한 문자열<sub>string</sub>을 지정하는 방법이 있습니다. 그리고 문자열 대신 호출 가능한 함수<sub>callable</sub>의 형태로 직접 지정하는 방법이 있습니다." 로 풀어쓰도록 하겠습니다.

File: sources/initializers.md [hunk 11]
```diff
@@ -1,394 +1,277 @@
...
 ```python
 keras.initializers.VarianceScaling(scale=1.0, mode='fan_in', distribution='normal', seed=None)
 ```
+가중치 텐서의 규모에 따라 초기값 생성에 사용할 분포를 조정합니다.  
 
-가중치의 형태에 따라 규모를 조절할 수 있는 초기값 설정기입니다.
-
-`distribution="normal"`로 분포를 설정 시, 0을 중심으로 `stddev = sqrt(scale / n)`의
-표준편차를 가진 절단된 정규분포에 따라 샘플이 생성되는데, 여기서 n이란:
+먼저 정규분포(`distribution="normal"`)를 선택할 경우, `0`을 평균으로 `sqrt(scale / n)`의 표준편차를 가진 정규분포의 `+-2`표준편차 범위 안에서 값을 선택합니다. 이때 `n`값은 `mode`인자 지정에 따라 다음과 같이 달라집니다.
```
[the remark this answers is not in the document]
Comment (EthanJYK) [reply sources/initializers.md]: 코멘트 보고 코드를 확인했습니다. 일단 정확한 값은 sqrt(scale/n) * 1/0.87962566103423978 을 표준편차로 하여 백엔드의 함수에서 +-2 표준편차를 잘라내는 TruncatedNormal을 적용하는 것입니다. 왜 * 1/0.87962566103423978 을 곱했을까 궁금해서 R로 코드를 돌려봤더니, Truncated된 분포에서의 표준편차를 Truncated 이전과 근사하게 유지하기 위해서 이렇게 한 것 같네요. 그러니까, 실제 작동방식은 처음에 제가 생각했던 내용인 "sqrt(scale/n)을 구해서 이 표준편차로 정규분포를 그린 후, +-2 영역을 잘라낸 분포에서 랜덤 생성을 한다(분포의 일부가 잘려나갔으므로 바뀐 분포의 표준편차도 바뀌죠)"가 아니라, "TruncatedNormal 형태의 분포로서, 바뀐 형태 자체로 sqrt(scale/n)의 표준편차를 갖는 분포에서 랜덤 생성을 한다"였습니다. 이 점 반영해서, "`0`을 평균으로 `sqrt(scale / n)`의 표준편차를 가진 절단 정규분포의 범위 안에서~"로 수정합니다.

**Comment by fuzzythecat** [comment]
감사합니다!
